client/src/components/Subscribe.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl'
+import { TextField, Button, Slide } from '@material-ui/core';
+
+const styles = {
+  root: {
+    width: '100%',
+    backgroundColor: '#f64141',
+    position: 'fixed',
+    height: '15%',
+    bottom: 0,
+  },
+  actionItem: {
+    color: 'white',
+    transition: 'all .3s ease-in-out',
+    '&:hover': {
+      textDecoration: 'none',
+      color: 'white',
+      transform: 'scale(1.1)',
+    },
+    '&$selected': {
+      color: 'white',
+      backgroundColor: 'red',
+    },
+  },
+  selected: {},
+  buttons: {
+    fontSize: '40px',
+
+    transition: 'all .3s ease-in-out',
+    '&:hover': {
+      transform: 'scale(1.1)',
+    },
+  },
+};
```

Comment:
Frankly, I haven't been looking to see if other components have been following this, but we should extract all CSS into a related `Subscribe.css` file instead of inline with the JS

Reply:
I'll look into this